skore/tests/unit/sklearn/comparison/cross_validation/test_report_metrics.py

```diff
@@ -0,0 +1,199 @@
+"""Tests of `ComparisonReport.metrics.report_metrics`."""
+
+import pandas as pd
+import pytest
+from pandas.testing import assert_frame_equal, assert_index_equal
+from sklearn.datasets import make_classification, make_regression
+from sklearn.dummy import DummyClassifier, DummyRegressor
+from skore import ComparisonReport, CrossValidationReport
+
+
+@pytest.fixture
+def classification_data():
+    X, y = make_classification(class_sep=0.1, random_state=42)
+    return X, y
+
+
+def make_classifier():
+    return DummyClassifier(strategy="uniform", random_state=0)
+
+
+@pytest.fixture
+def report(classification_data):
+    X, y = classification_data
+
+    report = ComparisonReport(
+        [
+            CrossValidationReport(make_classifier(), X, y),
+            CrossValidationReport(make_classifier(), X, y, cv_splitter=3),
+        ]
+    )
+
+    return report
+
+
+@pytest.fixture
+def report_regression():
+    X, y = make_regression(random_state=42)
+
+    report = ComparisonReport(
+        [
+            CrossValidationReport(DummyRegressor(), X, y),
+            CrossValidationReport(DummyRegressor(), X, y, cv_splitter=3),
+        ]
+    )
+
+    return report
+
+
+def test_different_split_numbers(report):
```

Comment:
Actually, it would be great to a small description for each to know exactly what we test. It is usually self explicit but sometimes it is nice to know that we got some shortcuts.